python-checks/src/test/resources/checks/confusingTypeChecking/nonCallableCalled.py

```diff
@@ -4,25 +4,30 @@
 def empty_union(x: Union['A', 'B']):
   x()
 
-def foo(param1: int, param2: Set[int], param3: FrozenSet[int]):
+def foo(param1: int, param2: Set[int], param3: FrozenSet[int], param4: list[str], param5: set[int]):
   param1() # Noncompliant {{Fix this call; Previous type checks suggest that "param1" has type int and it is not callable.}}
   x = 42
   x() # OK, raised by S5756
 
-  param2() # Noncompliant
+  param2() # FN SONARPY-2022
```

Comment:
I would expect these FNs to be fixed after rebasing on the corresponding ticket.
Can you double check that this is fixed and update the test accordingly?